docs/core/integration-quality-scale/rules/test-before-setup.md

```diff
@@ -42,6 +42,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: MyIntegrationConfigEntry
     return True
 ```
 
+:::info
+Please note that this may also be implemented implicitly when using a data update coordinator via `await coordinator.async_config_entry_first_refresh()`
```

Comment:
```suggestion
Please note that this may also be implemented implicitly when using a data update coordinator via `await coordinator.async_config_entry_first_refresh()`.
```